source/about/mattermost-v9-changelog.md

```diff
@@ -1,11 +1,116 @@
+ - Emojis are now placed at cursor position while editing messages.
+ - Made keyboard shortcuts modal content DIV-accessible via the keyboard.
+ - Added Channel Bookmarks user interface (disabled by default and behind a feature flag).
+
+#### Administration
+ - Added a new feature where an admin with user management permission can now edit a user's settings.
+ - Added download functionality for admins to download server logs from **Server Logs** page in the **System Console**.
+ - LDAP vendor errors are now included in the Support Packet.
+ - Added metadata to the Support Packet.
+ - We are now adding the user's ID and session ID to the audit log's Actor field for the login event, to match what we provide for the logout event.
+ - Added support for custom status in bulk export/import.
+ - Marked the ``RemoteTeamId`` field of the ``RemoteCluster`` entity as deprecated.
+ - Added log ``Name`` and ``DisplayName`` of groups.
+ - Logged fields of users are now updated.
+
+#### Performance
+ - Added platform related information to the notification metrics.
+ - Added additional information to INP and LCP client metrics.
+ - Added minor performance improvements to webapp initialization.
+
+#### mmctl
+ - Added two new commands to mmctl, ``mmctl job list`` and ``mmctl job update``.
+ - Panic message is now printed when mmctl panics.
+ - Setting ``AdvancedLoggingJSON`` via mmctl is now allowed.
+
+### Bug Fixes
+ - Fixed an issue that displayed a wrong count for custom group members on the notification warning.
+ - Fixed a panic when the password was too long.
+ - Fixed an issue where configuration patches through mmctl did not correctly merge plugin configuration values.
+ - Fixed issues with the OpenID local development.
+ - Fixed an issue where Latex was not rendered in a code block as code when Latex rendering was disabled.
+ - Fixed an issue with saving custom roles.
+ - Fixed an issue with the left-hand side scrollbar auto-hide functionality for Chrome and Safari.
+ - Fixed Group Message to private channel conversion edge cases.
+ - Fixed an issue where users with the user management permission were unable to view the list of users in the **System Console > Users** page.
+
+### config.json
+New setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to all plans:
+ - Under ``ServiceSettings`` in ``config.json``:
+    - Added ``TerminateSessionsOnPasswordChange`` to configure the sessions revocation during password resets.
+
+#### Changes to the Enterprise plan:
+ - Under ``ElasticsearchSettings`` in ``config.json``:
+    - Added ``Backend`` to differentiate between Elasticsearch and Opensearch. The default value is Elasticsearch.
+
+### API Changes
+ - Added a new plugin API to create a common support packet.
+ - Added new API endpoints to manage remote clusters.
```

Comment:
https://github.com/mattermost/mattermost/pull/27465 got replaced by https://github.com/mattermost/mattermost/pull/27490. There is no longer a new plugin API endpoint.
```suggestion
 - Added new API endpoints to manage remote clusters.
```